src/envoy/http/backend_routing/filter_config.h

```diff
@@ -57,6 +58,14 @@ class FilterConfig : public Envoy::Logger::Loggable<Envoy::Logger::Id::filter> {
         throw Envoy::ProtoValidationException(
             "Path translation for BackendRouting rule must be specified", rule);
       }
+      if (rule.path_prefix() == Envoy::EMPTY_STRING) {
```

Comment:
can these be specified in proto validation rules?

Reply:
Beside,  coding wise, prefer   rule.path_prefix().empty()

Reply:
Good point. Done